diff -ura spotifyd.orig/Cargo.toml spotifyd.new/Cargo.toml
--- spotifyd.orig/Cargo.toml	2018-10-03 13:37:29.469984943 -0400
+++ spotifyd.new/Cargo.toml	2018-10-04 06:50:49.709965339 -0400
@@ -23,6 +23,7 @@
 rspotify = "0.2.3"
 chrono = "0.4"
 alsa = { version = "0.2", optional = true }
+openssl-probe = "0.1.2"
 
 [replace]
 "rust-crypto:0.2.36" = { git = "https://github.com/awmath/rust-crypto.git", branch = "avx2" }
diff -ura spotifyd.orig/src/main.rs spotifyd.new/src/main.rs
--- spotifyd.orig/src/main.rs	2018-10-03 13:37:29.469984943 -0400
+++ spotifyd.new/src/main.rs	2018-10-04 06:51:42.529964782 -0400
@@ -22,6 +22,7 @@
 extern crate tokio_io;
 extern crate tokio_signal;
 extern crate xdg;
+extern crate openssl_probe;
 
 use std::process::exit;
 use std::panic;
@@ -42,6 +43,9 @@
 mod dbus_mpris;
 
 fn main() {
+
+    openssl_probe::init_ssl_cert_env_vars();
+
     let opts = cli::command_line_argument_options();
     let args: Vec<String> = std::env::args().collect();
 
