diff -ura spotifyd.orig/Cargo.toml spotifyd.new/Cargo.toml
--- spotifyd.orig/Cargo.toml	2018-10-03 13:37:29.469984943 -0400
+++ spotifyd.new/Cargo.toml	2018-10-03 13:39:00.816650636 -0400
@@ -23,6 +23,7 @@
 rspotify = "0.2.3"
 chrono = "0.4"
 alsa = { version = "0.2", optional = true }
+openssl = {features = ["vendored"] }
 
 [replace]
 "rust-crypto:0.2.36" = { git = "https://github.com/awmath/rust-crypto.git", branch = "avx2" }
